  do not add -Wno-pointer-sign,
  kfreebsd-6 still uses gcc-3.4

--- src/share/mk/bsd.sys.mk
+++ src/share/mk/bsd.sys.mk
@@ -55,7 +55,7 @@
 CWARNFLAGS	+=	-Wno-uninitialized
 .  endif
 .  if !defined(WITH_GCC3)
-CWARNFLAGS	+=	-Wno-pointer-sign
+#CWARNFLAGS	+=	-Wno-pointer-sign
 .  endif
 . endif
 
